components/ui/Link.jsx

```diff
@@ -0,0 +1,41 @@
+
+export default function Link(props) {
+    return (
+      <>
+        <a
+         
+          className={` group flex  w-fit items-center gap-1.5 ${props.style}`}
+          href={props.link}
+        >
+          <p className={`font-Neufile text-2xl ${props.textStyle}`}>
+            {props.text}
+          </p>
+          <div className={`relative h-3.34 w-3.5 ${props.imageWrapperStyle}`}>
```

Comment:
I can't see where you defined the height and width in config file, You should define height and width in the config file.